src/Bijectors.jl

```diff
@@ -276,29 +277,27 @@ maporbroadcast(f, x::AbstractArray{<:Any,N}...) where {N} = map(f, x...)
 maporbroadcast(f, x::AbstractArray...) = f.(x...)
 
 # optional dependencies
```

Comment:
Can we also add the `using Requires` above and instead add here (in a separate `if`-statement!) a
```julia
if !isdefined(Base, :get_extension)
    using Requires
end
```
(Not sure, possibly you have to change the check below wrapping `__init__()` to `@static if !isdefined....` - at least this seems to be the common pattern)

Reply:
I think the `@static if` instead of `__init__` is only needed if we have anything else we want to put in `__init__`, no? In this case we don't, so we just put both `using Requires` and the definition of `__init__` inside the same branch?

Reply:
Apparently not :confused: Using `@static` now